Integrations/Panorama/Panorama.py

```diff
@@ -3272,6 +3278,331 @@ def panorama_get_traffic_logs_command():
+    if 'proto' in log:
+        pretty_log['IPProtocol'] = log['proto']
+    if 'recipient' in log:
+        pretty_log['Recipient'] = log['recipient']
+    if 'rule' in log:
+        pretty_log['Rule'] = log['rule']
+    if 'rule_uuid' in log:
+        pretty_log['RuleID'] = log['rule_uuid']
+    if 'receive_time' in log:
+        pretty_log['ReceiveTime'] = log['receive_time']
+    if 'sender' in log:
+        pretty_log['Sender'] = log['sender']
+    if 'sessionid' in log:
+        pretty_log['SessionID'] = log['sessionid']
+    if 'serial' in log:
+        pretty_log['DeviceSN'] = log['serial']
+    if 'severity' in log:
+        pretty_log['Severity'] = log['severity']
+    if 'src' in log:
+        pretty_log['SourceAddress'] = log['src']
+    if 'srcloc' in log:
+        pretty_log['SourceCountry'] = log['srcloc']
+    if 'srcuser' in log:
+        pretty_log['SourceUser'] = log['srcuser']
+    if 'sport' in log:
+        pretty_log['SourcePort'] = log['sport']
+    if 'thr_category' in log:
+        pretty_log['ThreatCategory'] = log['thr_category']
+    if 'threatid' in log:
+        pretty_log['Name'] = log['threatid']
+    if 'tid' in log:
+        pretty_log['ID'] = log['tid']
+    if 'to' in log:
+        pretty_log['ToZone'] = log['to']
+    if 'time_generated' in log:
+        pretty_log['TimeGenerated'] = log['time_generated']
+    if 'url_category_list' in log:
+        pretty_log['URLCategoryList'] = log['url_category_list']
+
+    return pretty_log
+
+
+def prettify_logs(logs):
+    if not isinstance(logs, list):  # handle case of only one log that matched the query
+        return prettify_log(logs)
+    pretty_logs_arr = []
+    for log in logs:
+        pretty_log = prettify_log(log)
+        pretty_logs_arr.append(pretty_log)
+    return pretty_logs_arr
+
+
+def panorama_get_logs_command():
+    job_id = demisto.args().get('job_id')
+    result = panorama_get_traffic_logs(job_id)
+    ignore_auto_extract = demisto.args().get('ignore_auto_extract') == 'true'
+    log_type_dt = demisto.dt(demisto.context(), f'Panorama.Monitor(val.JobID === "{job_id}").LogType')
+    if isinstance(log_type_dt, list):
+        log_type = log_type_dt[0]
+    else:
+        log_type = log_type_dt
+
+    if result['response']['@status'] == 'error':
+        if 'msg' in result['response'] and 'line' in result['response']['msg']:
+            message = '. Reason is: ' + result['response']['msg']['line']
+            return_error('Query logs failed' + message)
+        else:
+            return_error('Query logs failed')
+
+    query_logs_output = {
+        'JobID': job_id,
+        'Status': 'Pending'
+    }
+
+    if 'response' not in result or 'result' not in result['response'] or 'job' not in result['response']['result']\
+            or 'status' not in result['response']['result']['job']:
+        return_error('Missing JobID status in response')
+
+    if result['response']['result']['job']['status'] != 'FIN':
```

Comment:
@yuvalbenshalom If the logs aren't done, i'm updating the context with the JobID and the Pending status.